apps/onlyoffice_docker.sh

```diff
@@ -30,6 +30,8 @@ else
     # Remove config.php value set when install was successful
     nextcloud_occ config:system:delete allow_local_remote_servers
     nextcloud_occ config:system:delete onlyoffice
```

Comment:
@enoch85 this line should already delete the complete array so deleting each value manually should not be necessary...

Reply:
Hmm, strange because it didn't update for a customer I just made a reinstall with. Even if set in the end of the script. I had to run the whole thing with `-x` to see the secret and then paste it in the GUI.

Maybe we should `print_text_in_color` for the secret itself just so that it can be handy to copy from the output?

Reply:
Even if not removed, it should get overwritten with the new value here:
https://github.com/nextcloud/vm/blob/703fe1a3bbdc57c0f876c6767acecf01edf47304/apps/onlyoffice_docker.sh#L261-L262

So not sure what got wrong on your customers reinstall...

Reply:
>  Even if set in the end of the script

> it should get overwritten with the new value here:

Me neither. Let's leave it as is.